chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/CertificationBodyManagerImpl.java

```diff
@@ -122,6 +122,9 @@ public CertificationBodyDTO update(final CertificationBodyDTO acb)
     @Transactional
     @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).CERTIFICATION_BODY, "
             + "T(gov.healthit.chpl.permissions.domains.CertificationBodyDomainPermissions).RETIRE)")
+    @CacheEvict(value = {
```

Comment:
There is an "update" method in here that I think also needs the same CacheEvict added. Also, could you do the refactor on this manager where you get rid of the "Impl" class and just have the manager class?